dotcom-rendering/src/client/islands/whenIdle.ts

```diff
@@ -5,9 +5,10 @@
  * @param callback the function to execute once the browser is 'idle'
  */
 export const whenIdle = (callback: () => void): void => {
+	const timeout = 500;
```

Comment:
This can be extracted to the top-level of the module, it does not need to be defined within the function body.

Reply:
I thought about that but still added it in scope anyway done 2dc93b6a1e3cc720e5544e01928b40ad58a97d12

Reply:
Be mindful that the way you've done it will break the JSDoc comments!

Reply:
You mean leaving a line space before the variable?

Reply:
The jsdoc comment describing the function is no longer directly above it, so jsdoc won't tie them together and intellisence etc. won't work

Reply:
Oh ok got it! Done b56a4594a15922a042de50fe90fce443dacc54b3